engine/framework/master.go

```diff
@@ -737,9 +747,8 @@ func (m *DefaultBaseMaster) exitWithoutSetErrCenter(ctx context.Context, exitRea
 		m.masterMeta.ErrorMsg = err.Error()
 	}
```

Comment:
Please clear the `ErrorMsg` here if the input `err` is nil, or the old ErrorMsg will remain in the job meta table even the job is finished normally.

Reply:
PTAL @maxshuang